db/schema.rb

```diff
@@ -18,6 +18,7 @@
   # Custom types defined in this database.
   # Note that some types may not work with other database engines. Be careful if changing database.
   create_enum "account_status", ["ok", "syncing", "error"]
+  create_enum "user_role", ["admin", "member"]
```

Comment:
This could be from some stale migrations.  You can run `bin/rails db:migrate:reset` to rerun everything from a clean slate, then `rake demo_data:empty` to generate the dev user again

Reply:
Good catch - fixed this!